src/Files.App/Actions/Navigation/NewTabAction.cs

```diff
@@ -12,6 +12,8 @@ internal class NewTabAction : IAction
 
 		public HotKey HotKey { get; } = new(VirtualKey.T, VirtualKeyModifiers.Control);
 
+		public RichGlyph Glyph { get; } = new("\uE710");
```

Comment:
Is this the right icon?

Reply:
It's the same we were using before.
![6](https://user-images.githubusercontent.com/102259289/227216338-28c75aa1-f82e-439a-a211-817b3f021b1a.png)

Reply:
We had a custom icon which was removed when I got rid of the custom icon font.

Reply:
How can I get it?

Reply:
It's not available anymore, I think we should remove this for now.

Reply:
That would remove the icon from the `TabView` control as well. I'll remove it only from the context menu, if this is what you mean